src/main/java/com/parasoft/demoapp/dto/GlobalPreferencesResponseDTO.java

```diff
@@ -0,0 +1,71 @@
+package com.parasoft.demoapp.dto;
+
+import com.parasoft.demoapp.model.global.preferences.*;
+import io.swagger.v3.oas.annotations.Hidden;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+import java.util.TreeSet;
+
+@Data
+@NoArgsConstructor
+@Schema
+public class GlobalPreferencesResponseDTO {
+
+    @Hidden
+    private DataAccessMode dataAccessMode;
+    @Hidden
+    private String soapEndPoint;
+
+    private IndustryType industryType;
+
+    private WebServiceMode webServiceMode;
+
+    private String graphQLEndpoint;
+
+    private Boolean advertisingEnabled;
+
+    private Set<DemoBugEntity> demoBugs = new TreeSet<>();
+
+    private Boolean useParasoftJDBCProxy;
+
+    @Schema(description = "Only works when useParasoftJDBCProxy field is enabled. Default value is http://localhost:9080 if this field is not set.")
```

Comment:
Remove @Schema in GlobalPreferencesEntity since it GlobalPreferencesResponseDTO will replace it to export to API. Other properties as well.